### DISCUSSION TITLE
Login Screen design

### DISCUSSION THREAD

**Discussion by CubE135** [requarks/wiki#3030] [Help / Questions] [answered]
Hey, i just upgraded my Wiki.js installation and noticed, that the login Screen Design changed a lot.
Before it looked like this:
![Old Login](https://user-images.githubusercontent.com/6615285/72224503-4f725880-3549-11ea-9a9c-d8a889b55b74.png)

And now it looks like this:

![New Login](https://www.rosehosting.com/blog/wp-content/uploads/2020/11/steps-on-how-to-install-wiki.js-on-centos-8.png)

It it meant to be like that?
The old one looks way better :/

**Comment by NGPixel** [comment, marked as answer]
Yes and you can change the background image under Admin > Security.

**Reply by EHRETic** [reply]
IF possible, it would be great to have it like the "new page" design, with the nice moving background, but on the other side, v3.0 is almost there... maybe there is something nicer coming no? 😊
